.github/workflows/cd.yml

```diff
@@ -238,6 +238,17 @@ jobs:
           command: |
             bash ./.github/scripts/download-simpleauth.sh
 
+      - name: replace sideloading placeholders
+        run: bash .github/scripts/sideloading-replace.sh
+        env:
+          SIDELOADING_SERVICE_ENDPOINT: ${{ secrets.SIDELOADING_SERVICE_ENDPOINT }}
+          SIDELOADING_SERVICE_SCOPE: ${{ secrets.SIDELOADING_SERVICE_SCOPE }}
+
+      - name: commit change on local
+        run: |
+          git add .
+          git commit -m "chore: replace sideloading placeholders"
```

Comment:
it's better to commit in a specific path, such as packages/cli/src/..
Doing this ensures that other changes are not committed at the same time.

Reply:
good point, updated.